package.json

```diff
@@ -19,13 +19,13 @@
 		"build-client": "gulp",
 		"build-concurrently": "yarn install-if-deps-outdated && yarn clean && yarn concurrently 'yarn build-client' 'yarn build-php' 'yarn build-extensions' 'yarn build-search' 'yarn build-packages'",
 		"build-extensions": "webpack --config ./webpack.config.extensions.js",
-		"build-packages": "webpack --config ./webpack.config.packages.js",
+		"build-packages": "cd packages/lazy-images/ && npm run build && cd ../../",
```

Comment:
Any reason we use `npm` instead of `yarn`?

Reply:
Also the `cd ../../` shouldn't be necessary, yarn does its thing in a subshell.

If that's future-proofing for potentially having to build more packages in the future, it might be best to future-proof a little further and put the commands in a build script in `bin/`.

Reply:
> Any reason we use `npm` instead of `yarn`?

Oversight on my part, apologies. I'll change to `npm` :+1: Thanks for spotting :+1:

Reply:
> Also the `cd ../../` shouldn't be necessary, yarn does its thing in a subshell.

Good point, I'll remove it :+1: 

> If that's future-proofing for potentially having to build more packages in the future, it might be best to future-proof a little further and put the commands in a build script in `bin/`.

I did it indeed for future-proofing! If we end up building more packages like this, I'd prefer using `lerna`, much like we do in Calypso, since it allows running the same command across a number of packages. (That would also allow us dropping the `cd packages/lazy-images/` here.) See e.g. https://github.com/Automattic/wp-calypso/blob/8b3ef578098c061eb6838c516761822908e911cb/package.json#L95, which basically runs the `prepare` script across all packages listed in [`lerna.json`](https://github.com/Automattic/wp-calypso/blob/8b3ef578098c061eb6838c516761822908e911cb/lerna.json).

Reply:
29b950f5d9d0fee9b217008cfbb0b3ea20b7b657

Reply:
Aaand f9c48b83b to use `yarn` instead of `npm run`.